e2e-tests/utils.ts

```diff
@@ -42,6 +42,8 @@ export async function createRandomPage(page: Page) {
   await page.fill('[placeholder="Search or create page"]', randomTitle)
   // Click text=/.*New page: "new page".*/
   await page.click('text=/.*New page: ".*/')
+  // Wait for h1 to be from our new page
+  await page.waitForSelector(`h1 >> text="${randomTitle}"`, { state: 'visible' })
```

Comment:
This is not strictly related to this PR, but I was finding this utility function would often fail on my machine by finishing early, detecting the textarea from the previous page instead

Reply:
Could it be that the input for the page title is not refreshed? Maybe we should give the input component a `key`.
https://github.com/logseq/logseq/blob/master/src/main/frontend/components/page.cljs#L242
@andelf Any thoughts?